lib/config.js

```diff
@@ -6,34 +6,25 @@ const path = require('path')
 
 module.exports = {
   AWS_INSTANCE_CHECK_URL: 'http://169.254.169.254/latest/dynamic/instance-identity/document/'
-, BUF_LIMIT: process.env.BUF_LIMIT || 10000 // 10000 lines
-, COMPRESS: isNaN(process.env.COMPRESS) ? true : +process.env.COMPRESS
+, COMPRESS: isNaN(process.env.COMPRESS) || process.env.COMPRESS === '1'
 , DEFAULT_CONF_FILE: os.platform() !== 'win32' ? '/etc/logdna.conf' : path.join(process.env.ALLUSERSPROFILE, '/logdna/logdna.conf')
 , DEFAULT_HTTP_ERRORS: ['ECONNRESET', 'EHOSTUNREACH', 'ETIMEDOUT', 'ESOCKETTIMEDOUT', 'ECONNREFUSED', 'ENOTFOUND']
 , DEFAULT_LOG_PATH: os.platform() !== 'win32' ? '/var/log' : path.join(process.env.ALLUSERSPROFILE, '/logs')
-, DEFAULT_REQ_HEADERS: {'Content-Type': 'application/json; charset=UTF-8'}
-, DEFAULT_REQ_HEADERS_GZIP: {'Content-Type': 'application/json; charset=UTF-8', 'Content-Encoding': 'gzip'}
 , DEFAULT_WINTAIL_FILE: os.platform() !== 'win32' ? '/etc/winTail.ps1' : path.join(process.env.ALLUSERSPROFILE, '/logdna/winTail.ps1')
 , FILEQUEUE_PATH: os.platform() !== 'win32'
     ? (process.env.FILEQUEUE_PATH || '/tmp')
     : path.join(process.env.ALLUSERSPROFILE, (process.env.FILEQUEUE_PATH || '/tmp'))
-, FLUSH_INTERVAL: process.env.FLUSH_INTERVAL || 250 // 250 millisec
-, FLUSH_LIMIT: process.env.FLUSH_LIMIT || 5000 // 5000 lines
-, GZIP_COMPRESS_LEVEL: process.env.GZIP_COMPRESS_LEVEL || 2
+, FLUSH_INTERVAL: process.env.FLUSH_INTERVAL
+, FLUSH_LIMIT: process.env.FLUSH_LIMIT
 , HEALTHCHECK_INTERVAL: process.env.HEALTHCHECK_INTERVAL || 1800000 // 30 minutes
-, HTTP_KEEPALIVE: process.env.HTTP_KEEPALIVE || 60000 // 1 min
```

Comment:
@darinspivey, what are replacing these `HTTP_KEEPALIVE`, `MAX_SOCKETS`, `SOCKET_KEEPALIVE` in the node client?

Reply:
I'll have a look to see what can be passed through to the client.

Reply:
These aren't supported in the client, and honestly, I think we should go with the defaults for those things.  `agentkeepalive` might be switched out in the future, and these are specific to it.  This being a `major`, I think this is a safe move.

Reply:
Sounds good!